scheduler_test.go

```diff
@@ -846,3 +846,19 @@ func TestRunJobsWithLimit(t *testing.T) {
 	assert.Exactly(t, 1, j1Counter)
 	assert.Exactly(t, 1, j2Counter)
 }
+
+func TestErrorDuringJobCreation(t *testing.T) {
+	// error due to bad time format
+	badTime := "0:0"
+	s := NewScheduler(time.Local)
+	s.Every(1).Day().At(badTime).Do(func() {})
+	assert.Zero(t, len(s.jobs), "The job should be deleted if the time format is wrong")
+
+	// error due to the arg passed to Do() not being a function
+	s.Every(1).Second().Do(1)
```

Comment:
do we have a test for `Do`? I really like mapping tests directly to functions

Reply:
Come to think of it, this function (`TestErrorDuringJobCreation`) is actually testing `Do` function. So, I could rename this function `TestDo`.